README.md

```diff
@@ -15,26 +15,22 @@ The C# extension is powered by [OmniSharp](https://github.com/OmniSharp/omnishar
 
 ### **Important!** Breaking Changes as of 1.0.10
 
-* The C# extension now only supports [.NET Core RC2](https://blogs.msdn.microsoft.com/dotnet/2016/05/16/announcing-net-core-rc2/). It no longer supports .NET Core RC1 or ASP .NET 5 RC1.
-* **Support for .csproj projects is partially functional on OS X and Linux.** This will impact anyone doing .csproj development on OS X or Linux (e.g. Unity, Xamarin, etc.). We are working to address this in the near future. However, for now, you can use the [Legacy C# Support extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.omnisharp).
+* The C# extension now only supports [.NET Core RC2](https://blogs.msdn.microsoft.com/dotnet/2016/05/16/announcing-net-core-rc2/) and newer. It no longer supports .NET Core RC1 or ASP .NET 5 RC1.
```

Comment:
This should be changed. Obviously, we don't just support .NET Core RC2 any longer. I'm wondering if we should just remove this and the line you're adding. Instead of listing this as a breaking change, I wonder if it's better to add text somewhere above this that says that we officially support .NET Core 1.0 and not bother mentioning the breaking change?